src/olympia/shelves/tests/test_views.py

```diff
@@ -168,6 +178,37 @@ def test_only_enabled_shelves_in_view(self):
         )
         assert result['results'][1]['addons'][0]['type'] == 'extension'
 
+    # If we delete HeroShelvesView move all the TestHeroShelvesView tests here
+    def test_hero_shelves_in_response(self):
+        phero = PrimaryHero.objects.create(
```

Comment:
I read that as a single word "phero", like "fero". Not a comment to address. Just something I found funny. Same with "shero", below (is that like a female hero)?